spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/src/main/groovy/org/springframework/cloud/contract/verifier/plugin/ContractsCopyTask.groovy

```diff
@@ -16,18 +16,21 @@
 
 package org.springframework.cloud.contract.verifier.plugin
 
-import groovy.transform.CompileDynamic
+
 import groovy.transform.CompileStatic
 import groovy.transform.PackageScope
+import org.gradle.api.Action
+import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
-import org.gradle.api.internal.ConventionTask
-import org.gradle.api.logging.Logger
-import org.gradle.api.tasks.TaskAction
-import org.gradle.api.tasks.WorkResult
-
-import org.springframework.cloud.contract.verifier.config.ContractVerifierConfigProperties
+import org.gradle.api.Project
+import org.gradle.api.file.CopySpec
+import org.gradle.api.file.Directory
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.*
```

Comment:
Let's be explicit about the imports please

Reply:
Was automatically squashed by IDE. Will fix.